Dockerfile

```diff
@@ -70,22 +36,23 @@ RUN go get -u -v github.com/ramya-rao-a/go-outline && \
     go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct && \
     go get -u -v github.com/go-delve/delve/cmd/dlv && \
     go get -u -v github.com/rogpeppe/godef && \
-    go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs && \
+    go get -u -v github.com/uudashr/gopkgs/v2/cmd/gopkgs && \
     go get -u -v golang.org/x/tools/cmd/gotype && \
+    go get -u -v github.com/mdempsky/gocode && \
+    go get -u -v github.com/stamblerre/gocode && \
     GO111MODULE=on go get -v golang.org/x/tools/gopls@latest && \
     go build -o /go/bin/gocode-gomod github.com/stamblerre/gocode && \
     chmod -R 777 /go && \
-    apk del .build-deps && \
-    mkdir /.cache && chmod -R 777 /.cache && \
+    mkdir -p /.cache && chmod -R 777 /.cache && \
+    mkdir -p /usr/local/go && chmod -R 777 /usr/local/go && \
     cd /usr/local/go && wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.22.2
 
 ENV GOPATH /go
```

Comment:
looks like this ENV was already set in base image https://github.com/docker-library/golang/blob/11eef5dfbc243e2c172c0ed0a38f65c0c756486d/1.14/stretch/Dockerfile#L75